pkg/model/cluster.go

```diff
@@ -30,9 +30,10 @@ import (
 )
 
 type ClusterConfig struct {
-	LaunchDelay      time.Duration
-	TerminateDelay   time.Duration
-	NumberOfRequests uint
+	LaunchDelay             time.Duration
+	TerminateDelay          time.Duration
+	NumberOfRequests        uint
+	InitialNumberOfReplicas int
```

Comment:
Suggest `uint` to catch unexpected negative values.

Reply:
Did it in https://github.com/pivotal/skenario/pull/67/commits/dfd4d60851eea3b87cc9d7e4dc950b95bc92c495 commit.